eZ Platform/Admin UI/Field Type Edit/Requirements.md

```diff
@@ -0,0 +1,35 @@
+# Content Editing
+
+## Objective
+
+The objective (which may be supported by prototypes or examples), is to identify and decide on the best approach to implement CRUD (create, read, update and delete) operations on Field Types within the user interface(s) and APIs exposed by eZ Platform in eZ Platform v2.x.
+
+The underlying goals are:
+
+- Simplify how to create a new Field Type (wether it is for eZ Systems' own needs when creating Field Types in the core platform, or developers customizing the platform by creating custom Field Types)
+- Reduce risk of errors and simplify maintenance by limiting the number of places where UI behavior is defined
+- Increase speed for developers to develop and maintain Field Types with a similar efficiency level as what was offered by eZ Publish Legacy, or better.
```

Comment:
This works but is still very vague. My point was that, for existing users, we can not make it more difficult (which is where we are now...), I am totally happy to higher the requirement and say "significantly easier than in legacy". Still a bit vague though.